<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthorWidget</class>
 <widget class="QWidget" name="AuthorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_lastname">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_firstname">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_middlename">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Middle Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_alias">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Alias:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_comment">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>210</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
